envs/hydromt-delft3dfm.yml

```diff
@@ -34,4 +34,4 @@ dependencies:
   - pip:
       - git+https://github.com/Deltares/hydromt.git
       - hydrolib-core==0.5.2
```

Comment:
`>=0.5.1` (or 0.5.2), but maybe not fix with `==` ?

Also, the ci is now failing since this yml file states `python=3.9.*`, maybe it can be replaced with python>=3.9.* or does the environment then fail?